spec/grocer_spec.rb

```diff
@@ -95,7 +96,7 @@
         consolidated_cart = consolidate_cart(perfect_avocado_cart)
         apply_coupons(consolidated_cart, [avocado_coupon])
         found_item = find_item_by_name_in_collection(item_with_coupon_applied_name, consolidated_cart)
-        expect(found_item[:price]).to eq(2.50), "After applying a $5 for 2 coupon to avocadoes, the price per unit is 2.50"
+        expect(found_item[:price]).to eq(2.50)
```

Comment:
Why cut the error test?

Reply:
if there is no text, the error produces a
```
expected: 2.50
actual: nil
```

With the text, they get that message and can't see the actual result of their work. Could fixed with interpolation.